docs/installation/advanced-installation-options.rst

```diff
@@ -41,3 +41,6 @@ with your own certificates:
 
  sudo ./install-prestoadmin.sh <path_to_cert>/cacert.pem
 
+Upgrading Presto-Admin
+----------------------
+Upgrading to a newer version of Presto-Admin requires deleting the ``presto-admin-install`` directory and ``presto-admin`` script from the older installation and then installing the newer version of Presto-Admin. To install the newer version, follow the instructions in the installation section (see :ref:`presto-admin-installation-label`)
```

Comment:
you probably want to delete the whole /opt/prestoadmin directory since when you untar, it will create a directory again